packages/firebase_ui_auth/example/pubspec.yaml

```
@@ -21,13 +21,9 @@ environment:
   sdk: '>=2.16.0 <3.0.0'
 
 dependencies:
-  cloud_firestore: ^4.0.1
-  crypto: ^3.0.1
   cupertino_icons: ^1.0.2
-  firebase_auth: ^3.10.0
-  firebase_core: ^1.10.3
-  firebase_database: ^10.0.0
-  firebase_dynamic_links: ^5.0.1
+  firebase_auth: ^4.0.0
```

auth is 4.0.1. https://github.com/firebase/flutterfire/blob/master/packages/firebase_auth/firebase_auth/pubspec.yaml#L7
```suggestion
  firebase_auth: ^4.0.1
```